packages/core-components/src/components/ProgressBars/Gauge.tsx

```diff
@@ -125,9 +138,13 @@ export function Gauge(props: GaugeProps) {
         strokeColor={getColor({ palette, value: asActual, inverse, max })}
         className={classes.circle}
       />
-      <div className={classes.overlay}>
-        {isNaN(value) ? 'N/A' : `${asActual}${unit}`}
-      </div>
+      {hoverMessage && isHovering ? (
+        <div className={classes.hoveringMessage}>{hoverMessage}</div>
```

Comment:
Hm, any possibility of using Tooltip from material-ui instead? It's the standard look-and-feel and has all of the accessibility concerns addressed etc.

https://v4.mui.com/components/tooltips/